setup.py

```diff
@@ -20,12 +20,12 @@ def read(*path):
     author_email='mail@gocept.com',
     description='Creates and drops temporary databases for testing purposes.',
     long_description=(
-        read('README.txt') +
-        read('COPYRIGHT.txt') +
+        read('README.rst') +
+        read('COPYRIGHT.rst') +
```

Comment:
Typically, the copyright is not necessarily formatted as RST, but it does not hurt here I guess.